change/@fluentui-react-textarea-089c5471-7d76-4622-b1a9-70af572fb526.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Adding missing spacing to top+bottom padding, font family to large variant, and min-height to size variants.",
```

_nit:_ We should do this now per [this RFC](https://github.com/microsoft/fluentui/blob/master/rfcs/convergence/changelog-guidelines.md)

```suggestion
  "comment": "fix: Adding missing spacing to top+bottom padding, font family to large variant, and min-height to size variants.",
```